opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/sort/SortQueryBuilder.java

```diff
@@ -53,11 +58,46 @@ public SortBuilder<?> build(Expression expression, Sort.SortOption option) {
         return SortBuilders.scoreSort().order(sortOrderMap.get(option.getSortOrder()));
       }
       return fieldBuild((ReferenceExpression) expression, option);
+    } else if (expression instanceof FunctionExpression
+        && ((FunctionExpression)expression).getFunctionName().getFunctionName().equalsIgnoreCase(
+        BuiltinFunctionName.NESTED.name())) {
+
+      validateNestedArgs((FunctionExpression) expression);
+      String orderByName = ((FunctionExpression)expression).getArguments().get(0).toString();
+      // Generate path if argument not supplied in function.
+      ReferenceExpression path = ((FunctionExpression)expression).getArguments().size() == 2
```

Comment:
What is the generated path in this case?

Reply:
See the javadoc for the function `generatePath()` A field of value `message.info.blah` would have a path of `message.info` generated.